deploy-cluster-aws/fabfile.py

```diff
@@ -97,45 +97,94 @@ def upgrade_setuptools():
     sudo('pip3 install --upgrade setuptools')
 
 
-# Install RethinkDB
+# Prepare RethinkDB storage
 @task
 @parallel
-def install_rethinkdb():
-    """Installation of RethinkDB"""
-    with settings(warn_only=True):
-        # preparing filesystem
-        sudo("mkdir -p /data")
-        # Locally mounted storage (m3.2xlarge, but also c3.xxx)
+def prep_rethinkdb_storage(USING_EBS):
+    """Prepare RethinkDB storage"""
+    # Convert USING_EBS from a string to a bool
+    USING_EBS = (USING_EBS.lower() == 'true')
+
+    # Make the /data directory for RethinkDB data
+    sudo("mkdir -p /data")
+
+    # OLD: with settings(warn_only=True):
+    if USING_EBS:  # on /dev/xvdp
+        # See https://tinyurl.com/h2nut68
+        sudo("mkfs -t ext4 /dev/xvdp")
```

Comment:
I tried a few different device names and then used ssh to login to the instance and see what the device names were (using `lsblk`). I also checked on the AWS EC2 web console. The EBS volume always had the name I'd assigned. That might not always work for all operating systems or even all Linux distros, but apparently it does work for the version of Ubuntu that we use by default for AWS deployments.